### ISSUE TITLE
Run accessibility tests against known content so we can be sure to cover everything users may encounter

### ISSUE THREAD

**Issue by greg-does-weather** [weather-gov/weather.gov#598]
### Description

Currently our accessibility tests for location pages loads the live Nashville page and tests it. However, this only tests the components that are currently enabled for Nashville. For example, if Nashville has no weather alerts but our alerts component has accessibility problems, our tests will miss them. More frustratingly, they will suddenly begin to fail later when Nashville has an alert.

We should instead have a collection of test pages, built by Drupal but with predefined arrangements of content (to reflect the arrangements we expect to see on the live site: with alerts, without alerts, with a weather story, without, etc.).

### Acceptance criteria

- [ ] We have created the well-known testing pages in dev (they do not need to exist outside of dev)
- [ ] Accessibility tests have been updated to use the well-known testing pages

**Comment by colinmurphy01** [comment]
Since we are building out our hazard components this sprint and next, @greg-does-weather should we consider tackling over the next two sprints?

**Comment by greg-does-weather** [comment]
I honestly don't know. I'd be interested in whether @eric-gade has thoughts about how to do this, but our current components don't provide a way to set their values – only their locations. So I suspect building static-ish test pages is going to be substantial effort.

**Comment by greg-does-weather** [comment]
I began some tinkering with putting a proxy server in our dev environment that, by default, forwards requests on to the API, but it can easily be switched to instead serve data from local JSON files. That's in the [mgwalker/api-proxy branch](https://github.com/weather-gov/weather.gov/tree/mgwalker/api-proxy). The technical implementation of the proxy is essentially finished and we'd only need to add JSON data files to serve.

There are probably other ways of accomplishing this, too, so I don't want to spend any additional time building this out without talking to the team first.
